src/__tests__/useSpring.test.ts

```diff
@@ -0,0 +1,119 @@
+import { act, renderHook } from '@testing-library/react-hooks';
+import useSpring from '../useSpring';
+import { Spring } from 'rebound';
+
+// simulate Spring for testing
+const mockSetCurrentValue = jest.fn();
```

Comment:
Mocking and reimplementing the whole rebound.Spring. This implies that I'm testing implementation here rather than behavior, but it's little bit crazy to test the whole behavior as 1) it's quite difficult and 2) I would be re-testing Spring itself, that's why I'm focusing on mocking and reimplementing everything here.